src/main/java/com/chekanova/imagetool/controller/ImageController.java

```diff
@@ -41,7 +42,9 @@ public ResponseEntity<byte[]> process(@RequestParam MultipartFile file,
             produces = IMAGE_JPEG_VALUE)
     public ResponseEntity<byte[]> compare(@RequestParam MultipartFile file1,
                                           @RequestParam MultipartFile file2,
+                                          @RequestParam(defaultValue = "#ff0000") String frameColor,
```

Comment:
You can write something like @Valid "@Pattern(regexp = "you-pattern-here") @RequestParam" and you will not need method 'validateHex'

Reply:
Done